src/tools/kata-ctl/src/ops/check_ops.rs

```diff
@@ -5,14 +5,22 @@
 
 use crate::arch::arch_specific::get_checks;
 
-use crate::args::{CheckArgument, CheckSubCommand, IptablesCommand, MetricsCommand};
+use crate::args::{
+    CheckArgument, CheckSubCommand, IptablesCommand, MetricsCommand, TimeoutArguments,
+};
 
 use crate::check;
 
 use crate::ops::version;
 
 use crate::types::*;
 
+use std::sync::Mutex;
+
+lazy_static! {
+    pub static ref KATA_TIMEOUT_SECS: Mutex<u64> = Mutex::new(1);
```

Comment:
This is fine but I'm not sure we actually need a mutex here as the tool is "single shot" and not threaded or async.

Reply:
Agree, I dont think we need a mutex here. The kata-ctl code is not threaded, so this is not required.

Reply:
I modified the implementation, I used mutex because I wanted to safely access global variables, Now I pass timeout value as a parameter into the function